elements/nuxeo-workflow-graph/nuxeo-workflow-graph.html

```diff
@@ -264,7 +268,7 @@
       },
 
       _fetchGraph: function() {
-        if (this.workflowId && this.workflowId.length > 0) {
+        if (this.taskId && this.workflowId && this.workflowId.length > 0) {
```

Comment:
I guess we acn cleanup the `.length > 0` check, since it will not be reached if `this.workflowId` is an empty string anyway:
```suggestion
        if (this.taskId && this.workflowId) {
```

Reply:
Nice catch, thanks!